feature-registry.csv

```diff
@@ -0,0 +1,126 @@
+# New features should be added to the bottom to not disrupt existing numbering.

```

Comment:
Where does the "on or off by default" data come from?

Reply:
This is a combination of John's list of default on features, the set of default features from harfbuzz subset, plus the features that Kahled mentioned being needed for math layout. I've added a comment to this file that documents that source of the data.